frontend/src/libraries/explorer-wamp/accounts.ts

```diff
@@ -97,4 +116,37 @@ export default class AccountsApi extends ExplorerApi {
   async queryAccount(id: string): Promise<any> {
     return await this.call<any>("nearcore-view-account", [id]);
   }
+
+  async queryLockupAccountInfo(accountId: string): Promise<any> {
+    // copied from https://github.com/near/near-wallet/blob/f52a3b1a72b901d87ab2c9cee79770d697be2bd9/src/utils/wallet.js#L601
+    const lockupAccountId =
+      sha256(Buffer.from(accountId)).substring(0, 40) +
+      "." +
+      LOCKUP_ACCOUNT_ID_SUFFIX;
+    try {
+      const lockupAccount = await this.queryAccount(lockupAccountId);
+      if (lockupAccount) {
+        const account = new nearAPI.Account({ provider: nearRpc });
```

Comment:
need your attention to failed test which is here.

Reply:
Ah, I see that is a TypeScript error. Pass an empty string as a second argument there, it is fine.

Reply:
<img width="716" alt="image" src="https://user-images.githubusercontent.com/33007476/96171495-d6d66900-0ed9-11eb-87c1-49d658cb4698.png">